examples/bind/live-list.amp.html

```diff
@@ -0,0 +1,257 @@
+<!doctype html>
```

Comment:
Please remove unnecessary pieces of this example page so that just the essential bind/live-list parts remain.